charts/oes/values.yaml

```diff
@@ -257,6 +272,11 @@ platform:
       - deployment_verification
       - sapor
       - visibility
+    oes:
+      approvalGate:
+        apiUrl: https://OES_GATE_IP:8084/visibilityservice/v2/approvalGates/{id}/trigger
+      verificationGate:
+        apiUrl: https://OES_GATE_IP:8084/autopilot/api/v2/registerCanary
```

Comment:
Let's define OES-UI url and OES-Gate url at a single place and use them all over. Not define it under every section

Reply:
Done